src/style.css

```diff
@@ -1,3 +1,64 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap");
+
 body {
   margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+  background-color: black;
+  display: flex;
+  justify-content: center;
+}
+
+.header {
+  width: 1200px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background-color: white;
+  padding: 0 50px;
+}
+
+.photo {
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
+  align-items: center;
+  padding: 0;
+}
+
+.nav__item {
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: normal;
+  text-transform: uppercase;
+  margin-left: 20px;
+  padding: 7px 0;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+a {
+  text-decoration: none;
+  color: #000;
+  font-size: 12px;
+  box-sizing: border-box;
+  padding: 23px 0;
+}
+
+a.is-active {
+  color: #00acdc;
+  border-bottom: 4px solid #00acdc;
+  border-radius: 2px;
+  padding-bottom: 19px;
+}
```

Comment:
The blue line under selected tab should be done with a pseudo element (like `after`). Please try it that way ;)

Reply:
Btw I don't think that border-radius is necessary